src/main/java/com/microsoft/sqlserver/jdbc/Parameter.java

```diff
@@ -505,34 +502,35 @@ private void setTypeDefinition(DTV dtv) {
                             // so, here, if the decimal parameter is encrypted and it is null and it is not outparameter
                             // then we set precision as the default precision instead of max precision
                             if (!isOutput()) {
-                                param.typeDefinition = "decimal(" + SQLServerConnection.defaultDecimalPrecision + ", "
-                                        + scale + ")";
+                                param.typeDefinition = SSType.DECIMAL.toString() + "("
+                                        + SQLServerConnection.defaultDecimalPrecision + ", " + scale + ")";
```

Comment:
I think this was a typo in the original implementation, and it should be `","` instead of `", "`. I don't see the extra space in any other param.typeDefinition.